src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFANewModifyPage.java

```diff
@@ -0,0 +1,128 @@
+    private WebElement accountSiteTextField;
+
+    @FindBy(how = How.ID, using = "acc6")
+    private WebElement typeList;
+
+    @FindBy(how = How.ID, using = "acc7")
+    private WebElement industryList;
+
+    @FindBy(how = How.ID, using = "acc8")
+    private WebElement annualRevenueTextField;
+
+    @FindBy(how = How.ID, using = "acc9")
+    private WebElement ratingList;
+
+    @FindBy(how = How.ID, using = "acc10")
+    private WebElement phoneTextField;
+
+    @FindBy(how = How.ID, using = "acc11")
+    private WebElement faxTextField;
+
+    @FindBy(how = How.ID, using = "acc12")
+    private WebElement websiteTextField;
+
+    @FindBy(how = How.ID, using = "acc13")
+    private WebElement tickerSymbolTextField;
+
+    @FindBy(how = How.ID, using = "acc14")
+    private WebElement ownershipList;
+
+    @FindBy(how = How.ID, using = "acc15")
+    private WebElement employeesTextField;
+
+    @FindBy(how = How.ID, using = "acc16")
+    private WebElement sicCodeTextField;
+
+    @FindBy(how = How.CSS, using = ".pbButton > input[name='save']")
+    private WebElement saveNewAccountButton;
+
+    /**
+     * Constructor.
+     */
+    public SFANewModifyPage() {
+        super();
+        PageFactory.initElements(this.webDriver, this);
+    }
+
+    /**
+     * Getter of saveNewAccountButton.
+     * @return button.
+     */
+    public WebElement getSaveNewAccountButton() {
+        return CommonActions.getElement(this.saveNewAccountButton);
+    }
+
+    /**
+     * Method to set accountName attribute.
+     * @param accountName string value.
+     */
+    public void setAccountNameTextField(final String accountName) {
+        this.accountNameTextField.sendKeys(accountName);
+    }
+    public void setParentAccountTextField(final String parentAccount) {
+        this.parentAccountTextField.sendKeys(parentAccount);
+    }
+    public void setAccountNumberTextField(final String accountNumber) {
+        this.accountNumberTextField.sendKeys(accountNumber);
+    }
+    public void setAccountSiteTextField(final String accountSite) {
+        this.accountSiteTextField.sendKeys(accountSite);
+    }
+    public void setAccountTypeList(final String type) {
+        this.typeList.sendKeys(type);
+    }
+    public void setIndustryList(final String industry) {
+        this.industryList.sendKeys(industry);
+    }
+    public void setAnnualRevenueTextField(final String annualRevenue) {
+        this.annualRevenueTextField.sendKeys(annualRevenue);
+    }
+
+    /**
+     * Method to click button.
+     */
+    public void clickSaveNewAccountButton() {
+        getSaveNewAccountButton().click();
+    }
+
+    public Map<String, Value> fillMethodsToFields() {
+        NEW_ACCOUNTS_FIELDS.put("accountName", (String v) -> {setAccountNameTextField(v);});
+        NEW_ACCOUNTS_FIELDS.put("parentAccount", (String v) -> {setParentAccountTextField(v);});
+        NEW_ACCOUNTS_FIELDS.put("accountNumber", (String v) -> {setAccountNumberTextField(v);});
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)